cfme/tests/services/test_orchestration_template.py

```diff
@@ -49,12 +49,17 @@ def pytest_generate_tests(metafunc):
     testgen.parametrize(metafunc, argnames, argvalues, ids=idlist, scope="module")
 
 
+def random_desc():
+    return fauxfactory.gen_alphanumeric()
+
+
 def test_orchestration_template_crud(provisioning):
     template_type = provisioning['stack_provisioning']['template_type']
     template = OrchestrationTemplate(template_type=template_type,
                                      template_name=fauxfactory.gen_alphanumeric(),
                                      description="my template")
-    template.create(METHOD_TORSO)
+    method = METHOD_TORSO.replace('CloudFormation', random_desc())
```

Comment:
If we are inserting a randomized string into the description, is there a need to have two ```METHOD_TORSO```s defined now? Also, as ```random_desc()``` does not hold any state etc, I don't see the real need to create a function for running that single line of code. I could be missing something though :)